src/test/java/org/wmn4j/notation/elements/KeyTest.java

```diff
@@ -0,0 +1,54 @@
+
+package org.wmn4j.notation.elements;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
```

Comment:
It is not necessary, but maybe this doc comment could include a brief description in addition to the author tag.

Reply:
I removed the author tag completely. I didn't add a description, it's probably self-evident what tests do?